### ISSUE TITLE
Make `Next Twine Node Interaction` take multiple TwineNodes, and update its name

### ISSUE THREAD

**Issue by grahamearley** [mjavaly/Prairie#83]
This might be more intuitive than having the user add multiple interactions for each potential Twine node.

**Comment by cpimhoff** [comment]
I can dig this - Potential issue with our desire to have the dropdown to choose a TwineNode (we'd have to nest the array GUI thing and our dropdown thing).

**Comment by grahamearley** [comment]
I'm actually thinking maybe "Associated Twine Node(s)" might be the best name for this (especially removing Interaction entirely). Really, what the user is doing is *associating* a TwineNode with a particular object that is interacted on.

Let's talk about this at our meeting today.

**Comment by grahamearley** [comment]
Multiple node input is in PR #110.

Still waiting on the rename though.

**Comment by grahamearley** [comment]
will move the rename part to a separate issue
